tags/resources/tag-formation-templates/template-roles.md

```diff
@@ -1,3 +1,103 @@
 # Roles within the `<TAG NAME>`
 
-*tbd* - contributions are welcome!
+The TAG includes several key roles that are critical to the group's success.
```

Comment:
```suggestion
<!--- **How to complete this template.**  For each area, select the appropriate roles and responsibilities that align with your TAG's expected operation and adjust the language as necessary. Not all TAGs will need or use all roles, or leverage the time limits suggested, and each TAG may choose to redefine or adjust the scope of responsibilities and activities for a given role. The TOC reserves the right to define what TAG Chairs are responsible for, a sample of which have been provided here. --->

The TAG includes several key roles that are critical to the group's success.
```

Reply:
this template will need aligned with the details provided here: https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#tag-formation-leadership-and-membership-composition

i would recommend linking that file within the template to assist TAGs in completing this role file after this PR has been reviewed and aligned with that document.

Reply:
the cncf-tags.md is out of date and needs updated. lets hold off on linking to it until its been modified.